packages/app/src/components/swatches/ColorSwatch.tsx

```diff
@@ -0,0 +1,77 @@
+import { RgbaColor } from 'noya-colorpicker/src/types';
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import styled from 'styled-components';
+import { memo } from 'react';
+
+
+const ColorSwatchContainer = styled.div(({ selected }: PropsSelected) => ({
```

Comment:
We should use theme colors here to support light/dark mode

Reply:
I think this can be updated to use more of the theme colors.  Maybe create a theme for this section?

Reply:
Sure - if you can get it pretty close with existing colors I can also tweak it a bit after we merge it too

Reply:
Okey! I think overall used it in the background and text. The exact ones I am no really sure